### PR TITLE
Add nutanix disconnected install behind proxy

### PR DESCRIPTION
**Pull request by sgaoshang**

### CODE REVIEW

**Comment by sgaoshang** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installer-rehearse-4.13-installer-rehearse-nutanix-ipi-proxy

**Comment by sgaoshang** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installer-rehearse-4.13-installer-rehearse-nutanix-ipi-proxy

**Comment by sgaoshang** [comment]
@jianlinliu pls help review nutanix disconnected install behind proxy, thanks.

File: ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse-4.13.yaml [hunk 1]
```diff
@@ -80,12 +80,12 @@ tests:
   steps:
     cluster_profile: vsphere-multizone
     workflow: cucushift-installer-rehearse-vsphere-ipi-zones
-- as: installer-rehearse-nutanix-ipi-disconnected
+- as: installer-rehearse-nutanix-ipi-proxy
```
Comment (jianlinliu) [inline ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse-4.13.yaml:83]: I suggest to not update the test name frequently, let use keep it like `installer-rehearse-nutanix-dis`, then you do some testing in the future, you just need to update workflow, no need to run `make jobs` every time.

File: ci-operator/step-registry/cucushift/installer/rehearse/nutanix/ipi/proxy/deprovision/cucushift-installer-rehearse-nutanix-ipi-proxy-deprovision-chain.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+chain:
+  as: cucushift-installer-rehearse-nutanix-ipi-proxy-deprovision
+  steps:
+  - ref: gather-mirror-registry
```
Comment (jianlinliu) [inline ci-operator/step-registry/cucushift/installer/rehearse/nutanix/ipi/proxy/deprovision/cucushift-installer-rehearse-nutanix-ipi-proxy-deprovision-chain.yaml:4]: I think it should be `ref: gather-proxy` here.

File: ci-operator/step-registry/cucushift/installer/rehearse/nutanix/ipi/proxy/provision/cucushift-installer-rehearse-nutanix-ipi-proxy-provision-chain.yaml [hunk 3]
```diff
@@ -0,0 +1,20 @@
+chain:
+  as: cucushift-installer-rehearse-nutanix-ipi-proxy-provision
+  steps:
+  - chain: ipi-conf-nutanix
+  - ref: ipi-install-rbac
+  - chain: nutanix-provision-bastionhost
+  - ref: mirror-images-by-oc-adm-in-bastion
```
Comment (jianlinliu) [inline ci-operator/step-registry/cucushift/installer/rehearse/nutanix/ipi/proxy/provision/cucushift-installer-rehearse-nutanix-ipi-proxy-provision-chain.yaml:7]: cluster install behind proxy also need the mirror operation? I think no.

File: ci-operator/step-registry/cucushift/installer/rehearse/nutanix/ipi/proxy/provision/cucushift-installer-rehearse-nutanix-ipi-proxy-provision-chain.yaml [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+  - ref: ipi-conf-proxy
+  - ref: openshift-cluster-bot-rbac
+  - ref: ipi-install-hosted-loki
+  - ref: ipi-install-install
+  - ref: ipi-install-times-collection
+  - ref: enable-qe-catalogsource-disconnected
+  - ref: set-sample-operator-disconnected
```
Comment (jianlinliu) [inline ci-operator/step-registry/cucushift/installer/rehearse/nutanix/ipi/proxy/provision/cucushift-installer-rehearse-nutanix-ipi-proxy-provision-chain.yaml:14]: I do not think we need the above two ref in cluster behind proxy, just need `ref: enable-qe-catalogsource`

File: ci-operator/step-registry/cucushift/installer/rehearse/nutanix/ipi/proxy/provision/cucushift-installer-rehearse-nutanix-ipi-proxy-provision-chain.yaml [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+  env:
+  - name: REGISTER_MIRROR_REGISTRY_DNS
```
Comment (jianlinliu) [inline ci-operator/step-registry/cucushift/installer/rehearse/nutanix/ipi/proxy/provision/cucushift-installer-rehearse-nutanix-ipi-proxy-provision-chain.yaml:16]: cluster install behind proxy does not need mirror registry.

File: ci-operator/step-registry/cucushift/installer/rehearse/nutanix/ipi/proxy/provision/cucushift-installer-rehearse-nutanix-ipi-proxy-provision-chain.yaml [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+  steps:
+  - chain: ipi-conf-nutanix
+  - ref: ipi-install-rbac
+  - chain: nutanix-provision-bastionhost
+  - ref: mirror-images-by-oc-adm-in-bastion
+  - ref: ipi-conf-proxy
+  - ref: openshift-cluster-bot-rbac
+  - ref: ipi-install-hosted-loki
+  - ref: ipi-install-install
+  - ref: ipi-install-times-collection
```
Comment (jianlinliu) [inline ci-operator/step-registry/cucushift/installer/rehearse/nutanix/ipi/proxy/provision/cucushift-installer-rehearse-nutanix-ipi-proxy-provision-chain.yaml:12]: I suggest to use "chain: ipi-install" here.

File: ci-operator/step-registry/cucushift/installer/rehearse/nutanix/ipi/proxy/provision/cucushift-installer-rehearse-nutanix-ipi-proxy-provision-chain.yaml (hunk 4 above)
Comment (jianlinliu) [reply ci-operator/step-registry/cucushift/installer/rehearse/nutanix/ipi/proxy/provision/cucushift-installer-rehearse-nutanix-ipi-proxy-provision-chain.yaml:14]: If possible, also pls help update `cucushift-installer-rehearse-nutanix-ipi-disconnected-provision` together.

File: ci-operator/step-registry/cucushift/installer/rehearse/nutanix/ipi/proxy/provision/cucushift-installer-rehearse-nutanix-ipi-proxy-provision-chain.yaml (hunk 3 above)
Comment (sgaoshang) [reply ci-operator/step-registry/cucushift/installer/rehearse/nutanix/ipi/proxy/provision/cucushift-installer-rehearse-nutanix-ipi-proxy-provision-chain.yaml:7]: Good catch, I missed remove it

File: ci-operator/step-registry/cucushift/installer/rehearse/nutanix/ipi/proxy/provision/cucushift-installer-rehearse-nutanix-ipi-proxy-provision-chain.yaml (hunk 6 above)
Comment (jianlinliu) [reply ci-operator/step-registry/cucushift/installer/rehearse/nutanix/ipi/proxy/provision/cucushift-installer-rehearse-nutanix-ipi-proxy-provision-chain.yaml:12]: If possible, also pls help update cucushift-installer-rehearse-nutanix-ipi-disconnected-provision together.

**Comment by sgaoshang** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installer-rehearse-4.13-installer-rehearse-nutanix-ipi-proxy

File: ci-operator/step-registry/cucushift/installer/rehearse/nutanix/ipi/proxy/provision/cucushift-installer-rehearse-nutanix-ipi-proxy-provision-chain.yaml (hunk 6 above)
Comment (sgaoshang) [reply ci-operator/step-registry/cucushift/installer/rehearse/nutanix/ipi/proxy/provision/cucushift-installer-rehearse-nutanix-ipi-proxy-provision-chain.yaml]: We need to run "- ref: ipi-install-rbac" before mirror steps, that's the reason why I split "chain: ipi-install", I'll check if install behind proxy still need it.
``` 
 - chain: nutanix-provision-bastionhost
  - ref: mirror-images-by-oc-adm-in-bastion
  `` `

Comment (jianlinliu) [reply ci-operator/step-registry/cucushift/installer/rehearse/nutanix/ipi/proxy/provision/cucushift-installer-rehearse-nutanix-ipi-proxy-provision-chain.yaml]: ah, got it.

**Comment by sgaoshang** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installer-rehearse-4.13-installer-rehearse-nutanix-ipi-proxy

**Comment by sgaoshang** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installer-rehearse-4.13-installer-rehearse-nutanix-ipi-dis

File: ci-operator/step-registry/cucushift/installer/rehearse/nutanix/ipi/proxy/provision/cucushift-installer-rehearse-nutanix-ipi-proxy-provision-chain.yaml (hunk 6 above)
Comment (sgaoshang) [reply ci-operator/step-registry/cucushift/installer/rehearse/nutanix/ipi/proxy/provision/cucushift-installer-rehearse-nutanix-ipi-proxy-provision-chain.yaml]: install behind proxy do not need run `- ref: ipi-install-rbac` beforehand, updated code

**Comment by sgaoshang** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installer-rehearse-4.13-installer-rehearse-nutanix-ipi-dis

**Comment by jianlinliu** [comment]
cc @liangxia to review.

**Pull request merged**